adagucserverEC/CXMLGen.cpp

```diff
@@ -1492,6 +1522,59 @@ int CXMLGen::getWCS_1_0_0_Capabilities(CT::string *XMLDoc, std::vector<WMSLayer
   return 0;
 }
 
+void CXMLGen::generateRangeSet(CT::string *XMLDoc, WMSLayer *layer) {
+  /*
+  From the documentation:
+  The optional and repeatable axisDescription/AxisDescription element is for compound observations.
+  It describes an additional parameter (that is, an independent variable besides space and time),
+  and the valid values of this parameter, which GetCoverage requests can use to select subsets of a
+  coverage offering.
+  */
+  XMLDoc->concat("    <rangeSet>\n"
+                 "      <RangeSet>\n"
+                 "        <name>dimensions</name>\n"
+                 "        <label>dimensions</label>\n");
+  // Dims
+  for (size_t d = 0; d < layer->dimList.size(); d++) {
+    WMSLayer::Dim *dim = layer->dimList[d];
+    if (dim->name.indexOf("time") != -1) {
```

Comment:
Would it be possible to show for time the min/max and resolution?

When I do describecoverage on a KNMI Radar file there is no time dimension in the describecoverage. I think we have to accomodate for a webclient, for a webclient it would be useful to have this information to generate a UI.

Reply:
The resolution in the same way that we show in the getCapabilities description?

Reply:
yes!